Source/Initialization/WarpXInitData.cpp

```diff
@@ -211,6 +213,65 @@ WarpX::ComputePMLFactors ()
     }
 }
 
+void
+WarpX::ComputeMaxStep ()
+{
+    if (do_compute_max_step_from_zmax) {
+        computeMaxStepBoostAccelerator(geom[0]);
+    }
+
+    max_step = std::min(max_step,
+                        istep[0] + static_cast<int>(std::ceil( (stop_time-t_new[0])/dt[0] )));
+}
+
+/* \brief computes max_step for wakefield simulation in boosted frame.
+ * \param geom: Geometry object that contains simulation domain.
+ *
+ * max_step is set so that the simulation stop when the lower corner of the
+ * simulation box passes input parameter zmax_plasma_to_compute_max_step.
+ */
+void
+WarpX::computeMaxStepBoostAccelerator(const amrex::Geometry& a_geom){
```

Comment:
That function was simply copy-pasted from `WarpXEvolve.cpp` to `WarpXInitData.cpp`.

Reply:
Thanks for the note, moved functions are always tricky to review :)